### PR TITLE
Lithium optimizations.

### PR DESCRIPTION
**Pull request by matt-42**

### CODE REVIEW

File: frameworks/C++/lithium/lithium.cc [hunk 1]
```diff
@@ -191,7 +190,7 @@ int main(int argc, char* argv[]) {
     std::sort(table.begin(), table.end(),
               [] (const fortune& a, const fortune& b) { return a.message < b.message; });
 
-    char b[100000];
+    char b[2000];
```
Comment (nbrady-techempower) [inline frameworks/C++/lithium/lithium.cc:193]: This isn't acceptable to cover the dynamic sizing rule of the fortunes test and won't pass the new verification step we're working on where we verify dynamic sizing by adding a large number of fortunes during that step

Comment (matt-42) [reply frameworks/C++/lithium/lithium.cc:193]: Ok. I'll work on a dynamic sizing implementation.

Comment (matt-42) [reply frameworks/C++/lithium/lithium.cc]: @nbrady-techempower This is fixed in my last commit.

Comment (nbrady-techempower) [reply frameworks/C++/lithium/lithium.cc]: thanks @matt-42!

**Pull request merged**